components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/s3/S3DatasetProperties_fr.properties

```
@@ -2,19 +2,19 @@ form.Main.title=Propri\u00E9t\u00E9s du jeu de donn\u00E9es S3
 form.Main.displayName=Propri\u00E9t\u00E9s du jeu de donn\u00E9es S3
 
 property.bucket.displayName=Bucket
-property.object.displayName=Chemin d''acc\u00E8s
-property.encryptDataInMotion.displayName=Chiffrer les donn\u00E9es \u00E0 la vol\u00E9e
+property.object.displayName=Chemin d'acc\u00E8s
```

I suggest to fix it:
```suggestion
property.object.displayName=Chemin d''acc\u00E8s
```